test/node-examples.js

```diff
@@ -130,10 +130,8 @@ async function runTestExample(folder) {
 
   await runTestExample('3d-heatmap');
   await runTestExample('arc');
-  await runTestExample('bezier');
```

Comment:
Shouldn't we still run the experimental examples?

Reply:
Disable them for now, will add a new PR to test those.